go/gilded-rose.go

```diff
@@ -13,6 +60,9 @@ func UpdateQuality(items []*Item) {
 				if items[i].name != "Sulfuras, Hand of Ragnaros" {
 					items[i].quality = items[i].quality - 1
 				}
+				if items[i].name == "Conjured Mana Cake" && items[i].quality > 0 {
```

Comment:
The requirements suggests that there could be many "Conjured" items. Personally I used strings to check If Conjured was part of the item name